src/main/java/teammates/common/util/EmailWrapper.java

```diff
@@ -0,0 +1,89 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents an email message and its important metadata.
+ */
+public class EmailWrapper {
+    
+    private String senderName;
+    private String senderEmail;
+    private String replyTo;
+    private List<String> recipientsList = new ArrayList<String>();
+    private List<String> bccList = new ArrayList<String>();
+    private String subject;
+    private String content;
```

Comment:
This list of attributes is far from exhaustive of everything that an email has, but has all the attributes needed for our current usage. It will be easy to extend if needed.